### PR TITLE
back to a working session key, never trust pocket js

### PR DESCRIPTION
**Pull request by nymd**

### CODE REVIEW

File: src/services/pocket-relayer.ts [hunk 1]
```diff
@@ -442,20 +443,24 @@ export class PocketRelayer {
...
+      // sessionKey = "blockchain and a hash of the all the nodes in this session, sorted by public key"
+      const sessionKey = `${blockchain}-${crypto
+        .createHash('sha256')
+        .update(
+          JSON.stringify(
+            nodes.sort((a, b) => (a.publicKey > b.publicKey ? 1 : b.publicKey > a.publicKey ? -1 : 0)),
+            (k, v) => (k !== 'publicKey' ? v : undefined)
+          )
+        )
+        .digest('hex')}`
```
Comment (Evalir) [inline src/services/pocket-relayer.ts:460]: This returns a string, right? Just to make extra extra sure

**Review by Evalir — DISMISSED** [review]
so sleek, so modern

File: src/services/pocket-relayer.ts (hunk 1 above)
Comment (nymd) [reply src/services/pocket-relayer.ts:460]: It did for the first 18 months it was in use in the codebase

**Review by crisog — APPROVED** [review]

**Review by Evalir — APPROVED** [review]
k now this looks good!

**Pull request merged**